### ISSUE TITLE
gRPC can't find binary for tools like check_epollexclusive.exe after 1.6.0 update

### ISSUE THREAD

**Issue by rsampaio** [Microsoft/vcpkg#1824]
When I try to build a project that was previously building with 1.4.x versions of gRPC I get this error: 

```
λ cmake -G "Visual Studio 15 2017 Win64" -DCMAKE_TOOLCHAIN_FILE=C:\Users\Rodrigo\Code\vcpkg\scripts\buildsystems\vcpkg.cmake ..
-- using protobuf
CMake Error at C:/Users/Rodrigo/Code/vcpkg/installed/x64-windows-static/share/grpc/gRPCTargets.cmake:173 (message):
  The imported target "gRPC::check_epollexclusive" references the file

     "C:/Users/Rodrigo/Code/vcpkg/packages/grpc_x64-windows-static/debug/bin/check_epollexclusive.exe"
```

I can confirm that the CMake target exported in 1.4 pointed to `tools/grpc/` path for these tools instead of `debug/bin`.

**Comment by a-andre** [comment]
See #1810.

**Comment by otsaregorodtsev** [comment]
Same thing for me. Had to manually copy

vcpkg\packages\grpc_x86-windows\tools\grpc\*.exe -> vcpkg\packages\grpc_x86-windows\debug\bin
vcpkg\packages\grpc_x86-windows\tools\grpc\*.exe -> vcpkg\packages\grpc_x86-windows\bin

**Comment by gwihlidal** [comment]
I get this error too. Here's the error output when trying to import grpc from vcpkg into my own project:

`CMake Error at External/vcpkg/installed/x64-windows/share/grpc/gRPCTargets.cmake:173 (message):
  The imported target "gRPC::check_epollexclusive" references the file

     "C:/halcyon/External/vcpkg/packages/grpc_x64-windows/debug/bin/check_epollexclusive.exe"

  but this file does not exist.  Possible reasons include:

  * The file was deleted, renamed, or moved to another location.

  * An install or uninstall procedure did not complete successfully.

  * The installation package was faulty and contained

     "C:/halcyon/External/vcpkg/installed/x64-windows/share/grpc/gRPCTargets.cmake"

  but not all the files it references.

Call Stack (most recent call first):
  External/vcpkg/installed/x64-windows/share/grpc/gRPCConfig.cmake:16 (include)
  External/vcpkg/scripts/buildsystems/vcpkg.cmake:145 (_find_package)
  Modules/Asset.Service/CMakeLists.txt:3 (find_package)

`

**Comment by ras0219-msft** [comment]
Thanks for reporting this!

It should be fixed in f527812e.
